data/settings.js

```diff
@@ -6651,7 +6651,7 @@ source IMAP server.
 	import_environment: {
 		default: `
 TZ=%{env:TZ} CORE_OUTOFMEM=%{env:CORE_OUTOFMEM} CORE_ERROR=%{env:CORE_ERROR}
-PATH=%{env:PATH}`,
+PATH=%{env:PATH} MALLOC_MMAP_THRESHOLD_=131072`,
```

Comment:
would be good to mention that systemd notify envvars are provided automatically as well. they are quite often pointlessly configured.

Reply:
added